models/public/retinaface-anti-cov/model.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  RetinaFace-Anti-Cov is a customized one stage face detector to help people protect
+  themselves from CovID-19. More details provided in the paper <https://arxiv.org/abs/1905.00641>
+  and repository <https://github.com/deepinsight/insightface/tree/master/RetinaFaceAntiCov>
+task_type: detection
+files:
+  - name: cov2.zip
+    size: 2265013
+    sha256: 618753c00da34544c6b3f50924f5676fdc729eeab45218afb2cd2b8d5f7a2143
+    source: https://www.dropbox.com/s/6rhhxsbh2qik65k/cov2.zip?dl=1
+postprocessing:
+  - $type: unpack_archive
+    format: zip
+    file: cov2.zip
+model_optimizer_args:
+  - --input_shape=[1,3,640,640]
+  - --input=data
+  - --input_model=$dl_dir/mnet_cov2-0000.params
+  - --input_symbol=$dl_dir/mnet_cov2-symbol.json
```

Comment:
Does the `--input_symbol` option actually do anything? IIRC, I specifically removed it from our MXNet models, because it was useless.

Reply:
Done